packages/apps/kadena-transfer/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "kadena-transfer",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "4.2.4",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^29.5.1",
+    "@types/node": "^16.18.27",
+    "@types/react": "^18.2.6",
+    "@types/react-dom": "^18.2.4",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
+  },
```

Comment:
Make sure to include the `_phase:...` scripts [example](https://github.com/kadena-community/kadena.js/blob/d7ea6eb7e343c79532abe791b25ade2d530a0863/packages/apps/kadena-docs/package.json#L6-L7)